docs/Versioning.md

```diff
@@ -26,6 +26,7 @@ For more details on ONNX Release versions, see [this page](https://github.com/on
 
 | ONNX Runtime release version | ONNX release version | ONNX opset version | ONNX ML opset version | Supported ONNX IR version | [Windows ML Availability](https://docs.microsoft.com/en-us/windows/ai/windows-ml/release-notes/)|
 |------------------------------|--------------------|--------------------|----------------------|------------------|------------------|
+| 1.6.0 | **1.7** down to 1.2 | 12 | 2 | 6 | Windows AI 1.5+ |
```

Comment:
should be opset 13

Reply:
fixed

Reply:
one more update - the Windows AI 1.5+ should be 1.6+